### DISCUSSION TITLE
Attach sauce labs url execution link to HtmlReporter

### DISCUSSION THREAD

**Discussion by baselalali** [webdriverio/webdriverio#7734] [Community]
Hallo Community,

I am using  HTML Reporter to output my test execution. After each test i get the job url for sauce labs ex link.  I would like to attach to the Html Reporter the url  

Currently i' am using the following 

` reporters: [
        'spec',
        [
            HtmlReporter,
            {
                debug: true,
                outputDir: './reports/html-reports/',
                filename: 'report.html',
                reportTitle: 'Test',
                showInBrowser: false,
                useOnAfterCommandForScreenshot: false
            }
        ]
    ],`

**Comment by christian-bromann** [comment]
@baselalali please raise this issue in the repository of the reporter. Thank you!

**Reply by baselalali** [reply]
Can you send me the link of repository of the reporter.

**Reply by christian-bromann** [reply]
I am not sure which one you are using, check the NPM page for it.

**Reply by baselalali** [reply]
I solved the problem.   by 👍  afterSession(config: WebdriverIO.Config, capabilities: WebDriver.DesiredCapabilities, specs: string[]) {
        console.info("Check out job at %s", capabilities.testobject_test_report_url);
    }
